SPECS/gh/gh.spec

```diff
@@ -0,0 +1,80 @@
+Summary:        GitHub�s official command line tool.
+Name:           gh
+Version:        2.13.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/Tools
+URL:            https://github.com/cli/cli
+Source0:        https://github.com/cli/cli/archive/refs/tags/v%{version}.tar.gz#/cli-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/cli/cli/archive/refs/tags/v%{version}.tar.gz -O cli-%%{version}.tar.gz
+#   2. tar -xf cli-%%{version}.tar.gz
+#   3. cd cli-%%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2021-04-26 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#
+#   NOTES:
+#       - You require GNU tar version 1.28+.
+#       - The additional options enable generation of a tarball with the same hash every time regardless of the environment.
+#         See: https://reproducible-builds.org/docs/archives/
+#       - For the value of "--mtime" use the date "2021-04-26 00:00Z" to simplify future updates.
+Source1:        %{name}-%{version}-vendor.tar.gz
+
+BuildRequires:  golang >= 1.17.1
+BuildRequires:  git
+BuildRequires:  bash-completion
+BuildRequires:  fish
+BuildRequires:  zsh
+Requires:       bash-completion
+Requires:       fish
+Requires:       zsh
```

Comment:
I am wondering why are we requiring `fish/zsh` ?
We could probably add a subpackage for `fish & fish-completion` ,  `zsh & zsh-completion`

Reply:
fish/vendor_completions.d/gh.fish and /zsh/site-functions/_gh are ~6KB each. No need to make two more sub-packages for 6KB files. Hence putting them in default package.

Reply:
Sure, my concern was not the `6k` extra files, but the actual requires of `fish` & `zsh` which would be more.
But given this is not a core package.. I guess LGTM :)